config/environments/test.rb

```diff
@@ -45,6 +45,9 @@
   config.action_mailer.asset_host = Figaro.env.mailer_domain_name
   config.action_mailer.default_options = { from: Figaro.env.email_from }
 
+  config.action_controller.asset_host = ENV['RAILS_ASSET_HOST'] if
+    ENV.key?('RAILS_ASSET_HOST')
```

Comment:
I find this formatting super weird with the condition on the next line. Does Rubocop get mad at

```ruby
if ENV.key?()
  config.action_controller.asset_host = ENV[]
end
```